theseus/embodied/kinematics/robot_model.py

```diff
@@ -40,3 +41,34 @@ def forward_kinematics(self, robot_pose: RobotModelInput) -> Dict[str, LieGroup]
 
     def dim(self) -> int:
         return 1
+
+
+class UrdfRobotModel(KinematicsModel):
+    def __init__(self, urdf_path: str):
+        self.drm_model = drm.DifferentiableRobotModel(urdf_path)
+
+    def forward_kinematics(self, joint_states: RobotModelInput) -> Dict[str, LieGroup]:
+        """Computes forward kinematics
+        Args:
+            joint_states: Vector of all joint angles
+        Outputs:
+             Dictionary that maps link name to link pose
+
+        (Note: differentiable robot model uses the xyzw quaternion convention,
```

Comment:
Maybe this comment about xyzw --> wxyz should be in the code (around line 67)? Note that from the point of view of a user reading this docstring in the documentation, this conversion happens under the hood.